db/migrate/20230512100427_change_foreign_key_for_posts.rb

```diff
@@ -0,0 +1,5 @@
+class ChangeForeignKeyForPosts < ActiveRecord::Migration[7.0]
+  def change
+    rename_column :posts, :user_id, :author_id
+  end
+end
```

Comment:
- [ ] you can create a migration like this one for likes and comment :+1:

Reply:
Hi @VICTOR-CHUKWU 
I discussed this issue with the previous reviewer also, I do according to the project requirment so you can check it the link for the instruction and ERD diagram
link](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/projects/creating_data_model_project_v1_2.md)
![blog_app_erd_v1_1](https://github.com/Habtamu210/Blog-App/assets/36001215/1166c7fc-7af9-4d41-8483-bac1cc843021)

Reply:
Well resolved ✔️ 

![image](https://github.com/Habtamu210/Blog-App/assets/38088818/2188e6fc-d329-4377-9ef0-16e4accd8d2c)